package.json

```diff
@@ -38,6 +48,7 @@
     "compression": "^1.7.3",
     "cookie-parser": "^1.4.1",
     "cookie-session": "^1.0.0",
+    "copyfiles": "^2.1.0",
```

Comment:
This looks like it should be a dev dep

Reply:
It can't unfortunately because we're triggering a build during `npm ci` when building the image.

Reply:
`npm ci` will still install dev deps unless you specify `npm ci --production`